src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -9,6 +9,17 @@ CHANGELOG
  * Add method `getSupportedTypes(?string $format)` to `NormalizerInterface` and `DenormalizerInterface`
  * Deprecate `MissingConstructorArgumentsException` in favor of `MissingConstructorArgumentException`
  * Deprecate `CacheableSupportsMethodInterface` in favor of the new `getSupportedTypes(?string $format)` methods
+ * The following Normalizer classes will become final in 7.0:
+   * `ConstraintViolationListNormalizer`
+   * `CustomNormalizer`
+   * `DataUriNormalizer`
+   * `DateIntervalNormalizer`
+   * `DateTimeNormalizer`
+   * `DateTimeZoneNormalizer`
+   * `GetSetMethodNormalizer`
+   * `JsonSerializableNormalizer`
+   * `ObjectNormalizer`
+   * `PropertyNormalizer`
```

Comment:
we need a similar message in UPGRADE-6.3.md

Reply:
added